src/core/utils/alert.ts

```diff
@@ -1,10 +1,10 @@
-import { showToast } from "@openmrs/esm-framework";
+import { showSnackbar } from "@openmrs/esm-framework";
 
 export function errorAlert(title: string, msg: string) {
-  showToast({
-    critical: true,
+  showSnackbar({
```

Comment:
we are not using this too

Reply:
I have seen them being imported somewhere, this one is being used here `import { errorAlert } from "../../core/utils/alert";`